pkg/stis/lib/cs1/doblev.c

```diff
@@ -284,18 +284,14 @@ double ccdbias    i: bias level to subtract if we can't get it from overscan
 	    }
 	}
 	if (too_few > 0) {
-	    printf ("Warning  (blevcorr) %d image line", too_few);
-	    if (too_few == 1)
-		printf (" has");
-	    else
-		printf ("s have");
-	    printf (" too few overscan pixels.\n");
+		trlwarn(MsgText, sizeof(MsgText), "(blevcorr) %d image line%s too few overscan pixels.",
```

Comment:
back to 4 space indents/tab

Reply:
I'm sorry for these. There are numerous areas where these replacements "damaged" lines with mixed indents. I tried to fix them here and elsewhere, however I think my editor's automatic fixups did whatever it wanted here. If tabs outweigh spaces, it pick tabs. If there are more spaces than tabs it'll choose spaces.

Typically it explodes when the code looks like this:

```if (no_curly_braces)
do_something(); // no indent, or at least not a visible one
else
do_something_else();
```

It tries to fix them after making a guess about which indent style matches the rest of the file.

Reply:
Yeah, looks like the problem is in the original code, not specifically your parts Joe.  Not worth worrying about for this PR, I thought it might be a quick fix for the parts like in cs1/do2d.c where there is a chunk of code where the only difference is the indent in the diffs page, but it looks like to do it properly would be a lot of work and overkill for this PR.

Reply:
Okay, sounds good. I wish we could reformat the entire codebase  to 4 spaces. So many files in ACS, for example, have a mix of tabs, 2(!!) spaces, and the initial indent is offset by 1, so every line is either "unindented" (1 space), or indented by 3 or 5 spaces.

My brain didn't like it.